profile/profile.go

```diff
@@ -201,6 +212,11 @@ func initProcedureList(profile *profctx.Profile) {
 	switch profile.ProfileType {
 	case REGISTER:
 		profile.Procedures = []common.ProcedureType{common.REGISTRATION_PROCEDURE}
+	case GUTI_REGISTER:
+		profile.Procedures = []common.ProcedureType{
+			common.REGISTRATION_PROCEDURE,
+			common.GUTI_REGISTRATION_PROCEDURE,
```

Comment:
Logically. Register, De-register and then Register with GUTI is what we are looking for.

Reply:
yes, i changed profile. please check